docs/manual_testing/testing_with_vms/windows/install_kolibri.rst

```diff
@@ -0,0 +1,47 @@
+Install Kolibri
+---------------
+
+.. tip:: Make a “baseline” snapshot of your VM prior to installing  Kolibri the first time
+
+After you’ve tuned and optimized your VM and installed browsers, make one **Snapshot** before you install Kolibri. This will be your baseline VM snapshot upon which you will install each Kolibri version and restore to it to install the next one.
+
+.. image:: ./install_kolibri_01.png
+
+Download minimal content
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Download this small testing channel, or others:
+
+   * ``nakav-mafak`` - Kolibri QA Channel (~250MB)
+
+2. Use the following command to create Kolibri users for testing.
+
+   .. code-block:: bash
+
+      kolibri manage generateuserdata
+   * `nakav-mafak` - Kolibri QA Channel (~180MB)
```

Comment:
@Lewiscowles1986 
My bad, I included too many lines in the suggestion, so now we have some repeated content.
Could you delete lines from 22-28? Thanks!